exam1.py

```diff
@@ -0,0 +1,31 @@
+import sys
+sys.stdin = open("input1.txt","r")
+T = int(input())
+dic={}
+dic1 = {}
+record = []
+list1=[]
+for i in range(T):
+    a = input()
+    dic[a]=""
+    record = list(map(int,input().split()))
+    for j in record:
+        if j not in dic1:
+            dic1[j] = 1
+        else:
+            dic1[j] += 1
+
+    b = sorted(dic1.values(),reverse=True)
+    c = b[0]
```

Comment:
```suggestion
    c = max(dic1.values())
```
최댓값을 구할 땐 `max` 함수 활용이 가능합니다!